contributors/devel/development.md

```diff
@@ -30,8 +30,9 @@ some of which may be incompatible in subtle ways, so we recommend
 
 Kubernetes is written in the [Go](http://golang.org) programming language.
 To build Kubernetes without using Docker containers, you'll need a Go
-development environment. Builds for Kubernetes 1.0 - 1.2 require Go version
-1.4.2. Builds for Kubernetes 1.3 and higher require Go version 1.6.0. If you
+development environment. Builds for Kubernetes 1.0 - 1.2 require Go version 1.4.2. 
+Builds for Kubernetes 1.3 and 1.4 require Go version 1.6.0. Builds for Kubernetes 1.5 
+and higher require Go version 1.7.0. If you
```

Comment:
I don't think the go patch version matters, and you probably want to use the latest, so I'd just include major & minor.